docs/release_documentation/reflexion_vinh.adoc

```diff
@@ -0,0 +1,11 @@
+Da ich das Modul Softwareengineering I bereits 2019 bestanden und mein Studium für einen Auslandsaufenthalt pausiert habe, kam ich am Anfang des Sommersemesters 2022 in die Gruppe E1 Ressourcenverwaltung hinein. Es galt sich also sich mit den Gruppenmitgliedern und der Aufgabenstellung vertraut zu machen. Schnell wurde klar, dass das Team noch Verstärkung in der Implementation brauchte, was auch mein persönlicher Wunsch war. Zusammen mit dem Team ging ich also die Aufgabenstellung durch und machte mich vertraut mit den Arbeitsprozessen, sowie dem vorhandenen Code. Da ich selbst nur freizeitlich und noch nie in einem Team programmiert habe, musste ich mich auch in Technologien wie Github oder Docker einarbeiten.
+
+In zweiwöchigen-, später einwöchigen Abständen, fanden Teammeetings statt, in der man sich über Probleme, Anregungen, etc austauschen konnte. Außerdem gab es noch eine Whatsappgruppe.
+Die Kommunikation innerhalb des Teams lief nach meinem Empfinden sehr gut, jedoch gab es Probleme mit dem Themenersteller. Da ich aber kaum, bis gar nicht mit diesem kommunizieren musste, kann ich das nicht beurteilen.
+
+Meine Hauptaufgabe bestand darin die vom Projektleiter erstellten Issues auf Github abzuarbeiten. Dort beschrieb er sehr detailliert was programmiert werden sollte, die Datei in der die Funktion geschrieben werden sollte, die Input und den Output, sowie eine kleine Skizze. Im Kopf geblieben ist mir noch, dass mein Formatierungstool für viel Verwirrung im Team gesorgt hat, da es die 4 Spaces zum einrücken gegen 2 Tabs getauscht hat. Optisch gab es zwar keine Veränderung, jedoch hat Github dies natürlich als Veränderung markiert und es war nicht eindeutig zu sehen, was jetzt am eigentlichen Quellcode verändert wurde.
```

Comment:
Ich bin nicht der Projektleiter :D aber keine ahnung wie man mich sonst betiteln könnte xD

Reply:
Herrn Werner aka programmer of doom

Reply:
xD